src/Mod/Draft/draftguitools/gui_lines.py

```diff
@@ -63,18 +63,21 @@ def GetResources(self):
                 'MenuText': QT_TRANSLATE_NOOP("Draft_Line", "Line"),
                 'ToolTip': QT_TRANSLATE_NOOP("Draft_Line", "Creates a 2-point line. CTRL to snap, SHIFT to constrain.")}
 
-    def Activated(self, name="Line", icon="Draft_Line"):
+    def Activated(self, name=QT_TRANSLATE_NOOP("draft","Line"), icon="Draft_Line", task_title=None):
```

Comment:
It's a detail, but it would perhaps be clearer to change:
`    def Activated(self, name=QT_TRANSLATE_NOOP("draft","Line"), icon="Draft_Line", task_title=None):`
To:
`    def Activated(self, name="Line", icon="Draft_Line", title=translate("draft", "Line):`
You could then do away with:
```
        if task_title is None:
        ...
```

Reply:
I think that will fail on a language change because of the way Python handles default arguments.